osbs/utils.py

```diff
@@ -201,14 +201,14 @@ def checkout_git_repo(git_uri, git_ref, git_branch=None):
 
 
 @contextlib.contextmanager
-def paused_builds(osbs, quota_name=None):
+def paused_builds(osbs, quota_name=None, ignore_quota_errors=False):
     try:
         logger.info("pausing builds")
-        osbs.pause_builds(quota_name=quota_name)
+        osbs.pause_builds(quota_name=quota_name, ignore_quota_errors=ignore_quota_errors)
```

Comment:
Right, its a good idea - exception may occur when quota json is being loaded etc.